audify/qa/nodes/text_quality.py

```
@@ -0,0 +1,569 @@
+    chapters: list[str] = state["chapters"]
+    chapter_titles: list[str] = state["chapter_titles"]
+
+    # Copy nested containers so we never mutate inbound state in place.
+    retry_budget = {k: dict(v) for k, v in state.get("retry_budget", {}).items()}
+    flags = {k: list(v) for k, v in state.get("flags", {}).items()}
+
+    pending_escalation: list[int] = []
+    clean_chapters: list[str] = []
+    clean_titles: list[str] = []
+
+    for i, (chapter_text, title) in enumerate(zip(chapters, chapter_titles)):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)